lib/install/validate/storage.go

```diff
@@ -39,29 +39,39 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 		v.NoteIssue(err)
 		return
 	}
+	if err != nil {
+		v.NoteIssue(err)
+	}
 
 	// provide a default path if only a DS name is provided
 	if imageDSpath.Path == "" {
 		imageDSpath.Path = input.DisplayName
 	}
 
-	v.NoteIssue(err)
 	if ds != nil {
 		v.SetDatastore(ds, imageDSpath)
 		conf.AddImageStore(imageDSpath)
 	}
 
+	if !v.targetDatastoreIsWriteable(ctx, ds) {
```

Comment:
We spoke about this in person but I'm going to leave a comment here just for context -- the current check looks only at datastores connected to the host the session is connected to. This may or may not be the imagestore, and it may or may not be the volumestore. Instead, my function finds all of the hosts in the computed cluster and compares those to the hosts that each of the user-specified datastores uses and makes sure that those datastores can each be written to by at least one host in the cluster.